kratos/processes/find_global_nodal_neighbours_for_entities_process.cpp

```diff
@@ -254,9 +254,10 @@ std::unordered_map<int, std::vector<int>> FindNodalNeighboursForEntitiesProcess<
     {
     public:
         typedef GlobalPointersVector<NodeType> value_type;
-        value_type gp_vector;
+        typedef GlobalPointersVector<NodeType> return_type;
```

Comment:
maybe we should start using `using` over `typedef` as afaik it is preferred since C++11

but not for this PR...